source/DG.Core/Orchestrators/IApplicationOrchestrator.cs

```diff
@@ -2,13 +2,14 @@
 {
     using System;
     using System.Collections.Generic;
+    using System.Reflection;
     using DG.Core.Model.Output;
 
     public interface IApplicationOrchestrator
     {
         IEnumerable<StateReport> GetInstanceState(string application, string instanceName);
 
-        object GetProperties(string application, string instanceName);
+        IEnumerable<PropertyInfo> GetSettingsProperties(string application, string instanceName);
```

Comment:
Please don't use PropertyInfo as output values, if we try to use this function in web api - we will get troubles: https://stackoverflow.com/questions/11690807/exception-raised-on-attempt-to-serialize-propertyinfo

Reply:
I would like to suggest DTO instead and map it inside